### PR TITLE
Automating JupyterLab version updates

### PR DESCRIPTION
**Pull request by mbektas**
- added script to update JupyterLab version
- updated instructions

fixes https://github.com/jupyterlab/jupyterlab_app/issues/220
fixes https://github.com/jupyterlab/jupyterlab_app/issues/242

### CODE REVIEW

File: Release.md [hunk 1]
```diff
@@ -20,9 +20,21 @@ If the JupyterLab version is not changing with the new JupyterLab App release th
...
-1. Update all the dependencies in [package.json](package.json) (packages with `@jupyterlab` scope and also `@jupyter-widgets/jupyterlab-manager` for ipywidgets). Versions of `@jupyterlab` packages can be found in the commits for the release tag at https://github.com/jupyterlab/jupyterlab/releases
+1. Update all `@jupyterlab` package dependencies in [package.json](package.json) using
+    ```bash
+    yarn set_jupyterlab_version <new-jlab-version>
+    ```
+    `<new-jlab-version>` must match a released JupyterLab version such as `3.1.13`. This command will update dependencies with `@jupyterlab` scope.
+
+2. Update `@jupyter-widgets/jupyterlab-manager` version in [package.json](package.json) for ipywidgets if a compatible newer version is available.
+
+3. Update `jupyterlab` and `ipywidgets` python package versions in [env_installer/construct.yaml](env_installer/construct.yaml)
```
Comment (krassowski) [inline Release.md:31]:
```suggestion
3. Update `jupyterlab` and `ipywidgets` python package versions in [env_installer/construct.yaml](env_installer/construct.yaml)

4. If new extension packages were added (or removed) add them to the `const extensions` in `src/browser/extensions/index.ts`
```

**Review by blink1073 — COMMENTED** [review]
Maybe add the handling of the index file to the script?

**Comment by mbektas** [comment]
> Maybe add the handling of the index file to the script?

Automating that part is quite hard since `package.json` usually has more `@jupyterlab` items than `extensions` list. We can detect if an extension in extensions list is not in package.json, but `yarn build` will usually fail in that case already. Please let me know if I am missing something.

**Comment by blink1073** [comment]
I would say include everything in [`jupyterlab.extensions`](https://github.com/jupyterlab/jupyterlab/blob/776fee997d4ed8596989872e6a11e7354cc7732e/jupyterlab/staging/package.json#L209) and maybe have an explicit exclude list for ones we don't want, to avoid situations like https://github.com/jupyterlab/jupyterlab_app/pull/245

**Comment by krassowski** [comment]
I guess the same goes for [`jupyterlab.mimeExtensions`](https://github.com/jupyterlab/jupyterlab/blob/776fee997d4ed8596989872e6a11e7354cc7732e/jupyterlab/staging/package.json#L248). I just noticed that we are missing `@jupyterlab/javascript-extension` (which would explain #242) and `@jupyterlab/vega5-extension`.

**Review by blink1073 — COMMENTED** [review]
This instruction needs updating

**Review by blink1073 — APPROVED** [review]
Thank you!

**Pull request merged**